src/pages/AddItem/styles.css

```
@@ -1,8 +1,42 @@
+.add-container {
+  border-radius: 25px;
+  height: 90vh;
+  width: 500px;
+}
+
 .add-form {
+  text-align: left;
   align-items: center;
   flex-direction: column;
   justify-content: center;
   width: fit-content;
   display: flex;
-  margin: 0 auto 30px;
+  margin: 30px auto 5px;
+  height: 60vh;
```

```suggestion
gap: 1rem;
```
Your form is a little tight. 👀